### PR TITLE
TW-444: Improve UI web build

### PR DESCRIPTION
**Pull request by nqhhdev**
- Issue:
  - #447 
  - #444 
 
- Resolved: 

  - 447:
  
  https://github.com/linagora/twake-on-matrix/assets/99852347/3a41e600-ae4e-4c91-9354-1f71406b595a
  - 444

  https://github.com/linagora/twake-on-matrix/assets/99852347/e8bbe772-3b06-46fe-8401-b8caaefeabb9

### CODE REVIEW

**Review by drminh2807 — APPROVED** [review]

File: lib/pages/contacts_tab/contacts_tab_body_view.dart [hunk 1]
```diff
@@ -61,16 +60,12 @@ class ContactsTabBodyView extends StatelessWidget {
                 children: contactsListSorted
                     .map<Widget>(
                       (contact) => InkWell(
+                        borderRadius: BorderRadius.circular(12),
```
Comment (dab246) [inline lib/pages/contacts_tab/contacts_tab_body_view.dart:63]:
```suggestion
 const BorderRadius.all(Radius.all(12)),
```

File: assets/images/ic_twake_image_logo.svg
Comment (hoangdat) [inline assets/images/ic_twake_image_logo.svg:1]: why delete? somewhere still use the icon?

**Review by hoangdat — APPROVED** [review]

File: assets/images/ic_twake_image_logo.svg
Comment (nqhhdev) [reply assets/images/ic_twake_image_logo.svg:1]: Don't worry, It's only used for web version

File: lib/pages/contacts_tab/contacts_tab.dart [hunk 2]
```diff
@@ -71,6 +75,52 @@ class ContactsTabController extends State<ContactsTab>
...
+    }
+  }
+
+  void goToDraftChat({
+    required BuildContext context,
+    required String path,
+    required PresentationContact contact,
+  }) {
+    if (contact.matrixId != Matrix.of(context).client.userID) {
+      Router.neglect(
```
Comment (sherlockvn) [inline lib/pages/contacts_tab/contacts_tab.dart:109]: what do this do ?

File: lib/pages/contacts_tab/contacts_tab_view.dart [hunk 3]
```diff
@@ -11,7 +12,12 @@ class ContactsTabView extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: ContactsAppBar(contactsController),
+      appBar: PreferredSize(
+        preferredSize: Size.fromHeight(
+          ResponsiveUtils().isMobile(context) ? 120 : 96,
```
Comment (sherlockvn) [inline lib/pages/contacts_tab/contacts_tab_view.dart:17]: have you check in tablet responsive ?

File: lib/pages/contacts_tab/contacts_tab.dart (hunk 2 above)
Comment (nqhhdev) [reply lib/pages/contacts_tab/contacts_tab.dart:109]: Screenshot 2023-08-11 at 09 46 28

File: lib/pages/contacts_tab/contacts_tab_view.dart (hunk 3 above)
Comment (nqhhdev) [reply lib/pages/contacts_tab/contacts_tab_view.dart:17]: Currently, we are support 2 breakpoint 
- Mobile
- Web and Tablet

**Review by sherlockvn — APPROVED** [review]

**Pull request merged**